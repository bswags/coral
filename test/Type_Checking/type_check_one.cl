#cordbconn
server="server";
user="server";
password="server";
port="server";
DBName="server";
type="server";
#enddbconn

#cordb
#enddb
void main() {

  int a = 3 + 3.0;

}
