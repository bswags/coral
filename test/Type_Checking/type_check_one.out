Traceback (most recent call last):
  File "Type_Checking/type_check_one.clx", line 6, in <module>
    setServer("server")
NameError: name 'setServer' is not defined
